.pylintrc

```diff
@@ -474,7 +474,7 @@ max-locals=15
 max-parents=7
 
 # Maximum number of public methods for a class (see R0904).
-max-public-methods=20
+max-public-methods=25
```

Comment:
Add a comment in the `test_plots` file instead. https://github.com/arviz-devs/arviz-plots/blob/main/tests/test_plots.py#L1